.github/workflows/pr-build-live-branch.yml

```diff
@@ -90,8 +90,8 @@ jobs:
                   path: unzips/woocommerce
                   retention-days: 30
 
-            - name: Inform Beta Download webhook
-              if: steps.prepare.outputs.plugin-data != '{}'
+            - name: Inform Beta Download webhook if this is an internal PR
+              if: steps.prepare.outputs.plugin-data != '{}' && ! github.event.pull_request.head.repo.fork
```

Comment:
Are you aware we have a script `isCommunityContributor` that can be used to determine internal versus community PRs?

Reply:
@samueljseay Yes, but the script applies labels when you use it. We really should separate the is community contributor logic from the label application logic. Rather than do that in this PR I opted for a 1 line change.

Reply:
😵‍💫 I don't know why that script was written like that to run the apply label. Thats disappointing.